server/graphql/v2/object/ContributorProfile.ts

```diff
@@ -10,5 +16,42 @@ export const GraphQLContributorProfile = new GraphQLObjectType({
       type: GraphQLAccount,
       description: 'The account that will be used to create the contribution',
     },
+    forAccount: {
+      type: GraphQLAccount,
+      description: 'The account that will receive the contribution',
+    },
+    totalContributedToHost: {
+      type: GraphQLAmount,
+      description: 'The total amount contributed to the host by this contributor',
+      args: {
+        inCollectiveCurrency: {
+          type: GraphQLBoolean,
+          defaultValue: false,
+          description: 'When true, the amount is converted to the currency of the collective',
+        },
+        since: {
+          type: GraphQLDateTime,
+          description: 'The date since when to calculate the total',
+        },
```

Comment:
```suggestion
        since: {
          type: GraphQLDateTime,
          description: 'The date since when to calculate the total. Defaults to the start of the current year.',
        },
```